registry.json

```diff
@@ -1,5 +1,19 @@
 [
   {
+    "ticker": "AMDLO",                                                       
+    "poolId": "2730ca79a60b3b3f29b709bd39c21d66cb2c83e5441c479f755d0a3d", 
+    "social": {       
+      "website": "https://armadillo-pool.org",
+      "twitter": "Armadillo_Pool", 
+      "instagram": "",
+      "youtube": "",
+      "facebook": ""
+    },
+    "telegram": "Zebra [AMDLO]",                                          
```

Comment:
Please change this to the username of an individual. This looks like it is the display name.

Reply:
All done. Thanks.